### PR TITLE
Added fastrope keybind

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
**When merged this pull request will:**
- Add a keybind to fastrope

### CODE REVIEW

File: addons/fastroping/XEH_postInit.sqf [hunk 1]
```diff
@@ -8,6 +8,16 @@
     [FUNC(fastRopeServerPFH), 0, _this] call CBA_fnc_addPerFrameHandler;
 }] call CBA_fnc_addEventHandler;
 
+// Keybinds
+["ACE3 Vehicles", QGVAR(fastRope), localize LSTRING(Interaction_fastRope), {
+    if ([ACE_player, vehicle ACE_player] call FUNC(canFastRope)) then {
```
Comment (PabstMirror) [inline addons/fastroping/XEH_postInit.sqf]: I think we might want the following (probably don't need the in-vehicle check, but can't hurt)
```
if ((vehicle ACE_player) == player) exitWith {false};
if (!([ACE_player, vehicle ACE_player, []] call EFUNC(common,canInteractWith))) exitWith {false};
```

Comment (BaerMitUmlaut) [reply addons/fastroping/XEH_postInit.sqf]: Didn't you mean `(vehicle ACE_player) == ACE_player`?

**Comment by dansta** [comment]
I don't mean to feature creep, but could it possibly be made so that helicopter pilots are able to initiate a fast roping sequence and force people out? There's always that guy in the back who can't find the menu option when scrolling or who hasn't set the fast roping keybind.

**Comment by BaerMitUmlaut** [comment]
No.

**Comment by PabstMirror** [comment]
oops, thanks bär

**Pull request merged**